storybook/shared/box.js

```diff
@@ -8,39 +8,34 @@ const typeMap = {
   B: 2,
   C: 3,
   D: 4,
+  E: 5,
 }
-export default class Box extends React.PureComponent {
-  static displayName = 'Box'
 
-  props: {
-    type: 'A' | 'B' | 'C' | 'D',
-    style?: Object,
-    children?: any,
-    value?: string,
-    margin?: Spacing,
-  }
-
-  render() {
-    const {
-      children,
-      margin = [0, 0.5, 1],
-      style,
-      type,
-      value = type,
-      ...props
-    } = this.props
-    return (
-      <Grid margin={margin} {...props}>
-        <Grid
-          dev={typeMap[type]}
-          padding={[1, 0]}
-          align="center"
-          justify="center"
-          style={style}
-        >
-          {children || value}
-        </Grid>
+export default function Box({
+  children,
+  margin = [0, 0.5, 1],
+  style,
+  type,
+  value = type,
+  ...props
+}: {
```

Comment:
This might be good to pull outside the function parameters. A bit hard to parse when its inlined like this.